docs/customization.rst

```diff
@@ -298,6 +298,9 @@ Configuration Options
             Note that if the ``toc.integrate`` feature is enabled, the left
             sidebar is instead scrolled for the local table-of-contents as
             described above.
+
+          .. note::
+              To enable the ``toc.follow`` feature, a `palette` must be set.
```

Comment:
We can instead warn about known undesirable behavior and link to the original issue ticket. 

```suggestion
          .. warning::
              When enabling the ``toc.follow`` feature,
             a `palette` must be set to avoid some known undesirable behavior.
             See `issue 380 <https://github.com/jbms/sphinx-immaterial/issues/380>`_.
```